src/zif_mustache.intf.abap

```diff
@@ -1,7 +1,7 @@
 interface zif_mustache
   public .
 
-  constants version type string value '2.0.0'. " Package version
+  constants version type string value '2.1.0'. " Package version
```

Comment:
Please don't adjust the versions, let it to me

Reply:
ok, sorry, I reverted the version change.

Reply:
sorry for the delay, I was awfully busy the last weeks...